firmware_mod/scripts/mqtt-control.sh

```diff
@@ -240,6 +240,13 @@ done
     "${TOPIC}/motion/mqtt_snapshot")
       /system/sdcard/bin/mosquitto_pub.bin -h "$HOST" -p "$PORT" -u "$USER" -P "$PASS" -t "${TOPIC}"/motion/mqtt_snapshot ${MOSQUITTOPUBOPTS} ${MOSQUITTOOPTS} -m "$(motion_mqtt_snapshot status)"
     ;;
+        
+    "${TOPIC}/motion/mqtt_snapshot GET")
```

Comment:
This doesn't have to do with motion.
Also GET is somewhat inconsistent with our existing style. 
How about /snapshot/take_image

Reply:
GET was to clarify this is a request that responds only via mqtt /snapshot/take_image may be misinterpreted as saving the snapshot to file as well as mqtt